iquip/apps/builder.py

```diff
@@ -221,18 +221,18 @@ def updateDateTimeEditState(self):
     def value(self) -> Optional[str]:
         """Overridden.
         
-        Returns the value of the dateTimeEdit in ISO format if it is enabled.
+        Returns the value of the dateTimeEdit in ISO format if the checkBox is enabled.
         Otherwise returns None.
         """
-        return self.dateTimeEdit.dateTime().toString(Qt.ISODate) if self.dateTimeEdit.isEnabled() \
-               else None
-
+        return (self.dateTimeEdit.dateTime().toString(Qt.ISODate) if self.checkBox.isChecked()
+                else None)
```

Comment:
I think just using the `if-else` statement would look better!
```python
if self.checkBox.isChecked():
    return self.dateTimeEdit.dateTime().toString(Qt.ISODate)
return None
```
Or is there any reason you want to use the ternary operator?

Reply:
Oh, I couldn't comp up with an idea to use `if-else` statement😅
I will apply it.